ghost/admin/app/styles/layouts/billing.css

```
@@ -42,6 +42,84 @@
+}
+
+.gh-billing-empty-copy p {
+    color: var(--midgrey);
+    font-size: 1.6rem;
+    line-height: 1.35;
+}
+
+.gh-billing-status-error a {
+    color: inherit;
+    padding-bottom: 1px;
+    border-bottom: 1px solid currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)